docs/admin/kubeadm.md

```diff
@@ -203,79 +260,124 @@ etcd:
   caFile: <path|string>
   certFile: <path|string>
   keyFile: <path|string>
-kubernetesVersion: <string>
 networking:
   dnsDomain: <string>
   serviceSubnet: <cidr>
   podSubnet: <cidr>
-secrets:
-  givenToken: <token|string>
-apiServerExtraArgs: {
-  <argument>: <value|string>,
-  <argument>: <value|string>,
-}
-controllerManagerExtraArgs: {
-  <argument>: <value|string>,
-  <argument>: <value|string>,
-}
-schedulerExtraArgs: {
-  <argument>: <value|string>,
-  <argument>: <value|string>,
-}
+kubernetesVersion: <string>
+cloudProvider: <string>
+authorizationMode: <string>
+token: <string>
+tokenTTL: <time duration>
+selfHosted: <bool>
+apiServerExtraArgs:
+  <argument>: <value|string>
+  <argument>: <value|string>
+controllerManagerExtraArgs:
+  <argument>: <value|string>
+  <argument>: <value|string>
+schedulerExtraArgs:
+  <argument>: <value|string>
+  <argument>: <value|string>
+apiServerCertSANs:
+  - <name1|string>
+  - <name2|string>
+certificatesDir: <string>
 ```
 
 ### Sample Node Configuration
 
 ```yaml
 apiVersion: kubeadm.k8s.io/v1alpha1
 kind: NodeConfiguration
-apiPort: <int>
-discoveryPort: <int>
-masterAddresses:
-- <master1>
-secrets:
-  givenToken: <token|string>
+caCertPath: <path|string>
+discoveryFile: <path|string>
+discoveryToken: <string>
+
+# Currently only the first server is used as a target for the cluster
+# bootstrap flow.
+discoveryTokenAPIServers:
+  - <address|string>
+  - <address|string>
+
+tlsBootstrapToken: <string>
+token: <string>
```

Comment:
I would leave token invisible here, if the config file is used, one should really specify the separate tokens.
token is more of a CLI-convenience kind of thing